server/config.ts

```diff
@@ -164,6 +164,15 @@ export default {
        */
       licencesUrl: get('LICENCES_URL', 'http://host.docker.internal:3000', requiredInProduction),
     },
+    frontendComponents: {
+      url: get('COMPONENT_API_URL', 'http://localhost:3000', requiredInProduction),
+      timeout: {
+        response: Number(get('COMPONENT_API_TIMEOUT_SECONDS', 10000)),
+        deadline: Number(get('COMPONENT_API_TIMEOUT_SECONDS', 10000)),
+      },
```

Comment:
can we change these default timeouts to 2000? 

Obviously this value was suggested, but long timeouts are a bit useless here. waiting 10 seconds to show the header is akin to an outage

Reply:
Yes, you are right.
I will changes the timeout to 2 seconds.